app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,12 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @use_posts = @user.recent_posts
```

Comment:
- [ ] Please 🙏🏽 correct this line in your controller to avoid errors in your view. It should be `@user_posts` 👇🏽 

![image](https://github.com/tomasesquivelgc/Blog-App/assets/58642949/e836e30c-d9cf-4056-a225-7bad7e8eb81d)

- [ ] When I opened the user#show page it should show just three posts, but it should all of them instead. Please correct the `recent_post` method and check that it shows just three posts.